belt-dwp/tests/belt.rs

```diff
@@ -0,0 +1,47 @@
+use aead::AeadInPlaceDetached;
+use belt_dwp::{BeltDwp, KeyInit};
+use hex_literal::hex;
+
+/// Test from Appendix A, tables 19-20 of [STB 34.101.31-2020](https://apmi.bsu.by/assets/files/std/belt-spec372.pdf)
```

Comment:
Since these doc strings do not get rendeed, you can write it like this:
```rust
/// Test from Appendix A, tables 19-20 of STB 34.101.31-2020:
/// https://apmi.bsu.by/assets/files/std/belt-spec372.pdf
```